python/amici/ode_export.py

```diff
@@ -1258,14 +1258,16 @@ def _generate_symbol(self, name: str) -> None:
             ])
             return
         elif name == 'dtcldp':
+            # check, whether the CL consists of only one state. Then,
+            # sensitivities drop out, syms can be deleted
+            state_set = {state.get_id() for state in self._states}
```

Comment:
shouldn't this be the same as `set(self.syms('x'))`?

Reply:
Maybe... will check. If so, your suggestions doesn't access private members directly and is hence nicer!